server/properties/src/config/logback.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2017 ThoughtWorks, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<configuration
+    xmlns="http://ch.qos.logback/xml/ns/logback"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://ch.qos.logback/xml/ns/logback http://ch.qos.logback/xml/ns/logback/logback.xsd"
+    debug="${gocd.server.logback.debug:-false}"
+    scan="${gocd.server.logback.scan:-true}"
+    scanPeriod="${gocd.server.logback.scanPeriod:-5 seconds}"
+>
+
+  <appender name="FileAppender" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${gocd.server.log.dir:-logs}/go-server.log</file>
```

Comment:
This will be messed up on linux. It should be /var/log/go-server in linux. People only configure the gocd.server.log.dir if they don't want the default.

Reply:
Remember the logic in LinuxPackagingTask to replace logs with /var/log/go-server..?

Reply:
I suppose you're hinting at setting `gocd.server.log.dir` somewhere in `server.sh`?

Reply:
Yep. If /var/log/go-server directory exists, then set this system property to /var/log/go-server